test/uk/gov/hmrc/statepension/domain/APIAccessSpec.scala

```diff
@@ -16,29 +16,31 @@
 
 package uk.gov.hmrc.statepension.domain
 
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.matchers.should.Matchers.shouldBe
 import play.api.libs.json.{JsSuccess, Json}
-import org.scalatest.matchers.must.Matchers.mustBe
 import org.scalatest.wordspec.AnyWordSpec
+import org.scalactic.Prettifier.default
 
-class APIAccessSpec extends AnyWordSpec {
+class APIAccessSpec extends AnyWordSpec with Matchers {
 
   "APIAccess" must {
     "serialize and deserialize correctly with whitelistedApplicationIds" in {
-      val apiAccess = APIAccess("PUBLIC", Some(Seq("app1", "app2")))
+      val apiAccess = APIAccess(`type` = "PUBLIC", whitelistedApplicationIds = Some(Seq("app1", "app2")))
```

Comment:
I recently took this out because it breaks the deployment to the API platform, sorry this can't go back in

Reply:
In fact, I don't even know how this compiled given https://github.com/hmrc/state-pension/blob/main/app/uk/gov/hmrc/statepension/domain/APIAccess.scala#L21

Reply:
Didn't have these changes within my branch, I've removed that spec file from the PR.